app/routes/slack.ts

```diff
@@ -0,0 +1,193 @@
+import { config } from '@config'
+
+import express from 'express'
+import { createEventAdapter } from '@slack/events-api'
+import { createMessageAdapter } from '@slack/interactive-messages'
+import { WebClient } from '@slack/web-api'
+
+const slackEvents = createEventAdapter(config.slack.signingSecret)
+const slackInteractions = createMessageAdapter(config.slack.signingSecret)
+
+const webClient = new WebClient(config.slack.token)
+
+const router = express.Router()
+
+router.use(async (req: any, __, next) => {
+  console.log(req.body)
+  next()
+})
+
+router.use('/events', slackEvents.expressMiddleware())
```

Comment:
I use `expressMiddleware` which allow to have node api and slack api on the same port 👌

Reply:
It init slackEvents listener